src/parser/core/modules/ActionWindow/windows/ActionWindow.tsx

```diff
@@ -137,7 +137,7 @@ export abstract class ActionWindow extends Analyser {
 		this.addEventHook('complete', this.onComplete)
 	}
 
-	private onComplete() {
+	protected onComplete() {
```

Comment:
This should return to private after making PlayersBuffedEvaluator do it's own suggestions.

```suggestion
	private onComplete() {
```